js/index.js

```diff
@@ -0,0 +1,24 @@
+//lesson 13
+const skillsSectiom = document.getElementById("skills")
+const skillsList = document.getElementById("skills-list")
+
+const skills = [ "HTML", "CSS","Javascript"];
+
+for(let i = 0; i < skills.length; i++){
+    let skill = document.createElement('li')
+    skill.textContent = `${skills[i]}`
```

Comment:
You could remove the template literals with the pound sign and brackets to interpolate value: `${}` because the value `skills[i]` is already a string and your not adding another piece of string inside the backticks.